### PR TITLE
Frontend edits - Course List Section

### LINKED ISSUE #151: Create initial Advising Dashboard frontend components

### PR DESCRIPTION
**Pull request by miaboloix**
Some additions to PR #155 
- Changed title on top bar
- Changed height of advising-schedule container
- Added box around course section

### CODE REVIEW

**Review by yesterd3v — APPROVED** [review]

**Comment by miaboloix** [comment]
Screen Shot 2021-03-30 at 12 42 43 PM

File: static/js/redux/ui/advising_schedule.jsx [hunk 1]
```diff
@@ -130,21 +130,16 @@ class AdvisingSchedule extends React.Component {
...
-            optionalSlots = null;
-        } else if (optionalSlots.length === 0) {
-            const img = (
-                <img
-                    src="/static/img/emptystates/optionalslots.png"
-                    alt="No optional courses added."
-                />);
-            optionalSlots = (
+        const waitlistedlSlotsHeader = (<div>
+            <h4 className="as-header">Wailisted Courses</h4>
```
Comment (JJamesWWang) [inline static/js/redux/ui/advising_schedule.jsx:134]: Typo here

**Review by JJamesWWang — APPROVED** [review]
lgtm

File: static/js/redux/ui/advising_schedule.jsx (hunk 1 above)
Comment (miaboloix) [reply static/js/redux/ui/advising_schedule.jsx]: fixed

**Pull request merged**